browser-sign-in/android/app/src/androidTest/java/com/browserSignIn/OktaSignInPageObject.kt

```diff
@@ -0,0 +1,44 @@
+package com.browserSignIn
+
+import android.webkit.WebView
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+
+class OktaSignInPageObject {
+
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private val selector = UiSelector()
+    private val timeout = 1000 * 60L
+
+    init {
+        device.wait(Until.findObject(By.clazz(WebView::class.java)), timeout)
```

Comment:
Can we use https://developer.android.com/training/testing/espresso/web instead?

I think it would provide the ability to not have to Thread.sleep, as well.

Reply:
We need to use ui automator since there is a separate process for sign-in widget view.